virttest/qemu_storage.py

```diff
@@ -986,9 +986,12 @@ def convert(self, params, root_dir, cache_mode=None,
         convert_cmd = self.image_cmd + " " + \
             self._cmd_formatter.format(self.convert_cmd, **cmd_dict)
 
-        LOG.info("Convert image %s from %s to %s", self.image_filename,
-                 self.image_format, convert_image.image_format)
-        process.run(convert_cmd)
+        timeout = params.get_numeric("conversion_timeout", 12000)
```

Comment:
Shall we use `convert_params` instead of `params` to get a fine-grained manipulation?

Reply:
IIRC my probelm with `convert_params` was that it wasn't exactly filtered version of params but a smaller set of parameters introduced locally. But even then what you say would make sense so I will check in more detail.